managed/services/inventory/grpc/nodes_server.go

```diff
@@ -106,6 +106,10 @@ func (s *nodesServer) GetNode(ctx context.Context, req *inventorypb.GetNodeReque
 	return res, nil
 }
 
+func (s *nodesServer) AddNode(ctx context.Context, req *inventorypb.AddNodeRequest) (*inventorypb.AddNodeResponse, error) {
+	return s.svc.AddNode(ctx, req)
+}
+
```

Comment:
Minor: I think we can remove this file at all. Some time ago we agreed that these `grpc` packages are redundant.

Reply:
Ok, we should remove all the three in one shot IMO. Will do as a separate effort.